sematic/plugins/building/docker_builder.py

```diff
@@ -35,6 +36,7 @@
     BuildConfigurationError,
     BuildError,
 )
+from sematic.plugins.building.docker_client_utils import _rolling_print_status_updates
```

Comment:
Since `_rolling_print_status_updates` is imported, it should be public (remove leading `_`).

Reply:
Done.